CHARTER.md

```diff
@@ -13,6 +13,17 @@ The mission of Aggie Robotics is to provide practical educational opportunities
 
 ## Membership
 
+Membership in Aggie Robotics contains 3 levels: Journeyman Membership, Associate Membership, and Full Membership. All members will receive a `Member Badge`. Members can advance in level by `badging`. The `badging` process will be defined by the by-laws and published openly.
```

Comment:
@NowYouCanFindMe I think that's a cool idea! I do think that all regulation of "badging" should take place in by-laws, as they will have to be updated much more often than things we will put in our charter. I recommend we bring this up when drafting those, after we have a charter and and organization.